libethereum/BlockQueue.cpp

```diff
@@ -131,6 +127,7 @@ void BlockQueue::verifierBody()
 				}
 			cwarn << "BlockQueue missing our job: was there a GM?";
 			OK1:;
+			drainVerified();
```

Comment:
what if the bad block, `m_verifying[0]`, was on a different chain (perhaps it's a sibling) to `m_verifying[1]`?